src/libcephfs.cc

```diff
@@ -34,6 +34,9 @@
 
 #include "include/cephfs/libcephfs.h"
 
+mode_t DEFAULT_UMASK = 022;
```

Comment:
Is this the correct?

Reply:
Yes, that's a suitable default.

Reply:
This should be a `#define DEFAULT_UMASK`. The variable will be stored in `struct ceph_mount_info`.

Reply:
Okay.